estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+estate_properry_model,access_model,model_estate_property,base.group_user,1,1,1,1
```

Comment:
Also, make sure to use a different name for each rule. You can even use the same as the ID.

```suggestion
estate_property_model,access_model,model_estate_property,base.group_user,1,1,1,1
```